src/middleware/notFound.middleware.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextFunction, Request, Response } from 'express';
+
+const routeNotFound = (req: Request, res: Response, next: NextFunction) => {
+  console.log('route not found');
+  next();
```

Comment:
As we are sending response directly from middle ware, no need to call next() function
```suggestion
  next();
```